### PR TITLE
Update Contributing.md

### PR DESCRIPTION
**Pull request by ari0211**
### 🛠️ Fixes Issue
#39 
Thank you @AbhijithGanesh for pointing it out!

### 👨‍💻 Changes proposed

Asking the contributors to make changes in the `dev` branch so as to remove any confusion.

### 🔧 Check List (Check the type of issue it fixes) 

- [ ] Bug Fix
- [ ] New Extension
- [ ] Enhancement
- [ ✔️] Documentation

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -40,9 +40,9 @@ git push -u origin "branch name"
 ```
 
 ## Create a PR
-
-Click on contribute and perfrom a Pull Request
-And also describe a bit what changes made and try to attach screenshots.
+First things first,dont forget to star the repo!
+Click on contribute and perfrom a Pull Request,and also describe a bit what changes you have made and try to attach screenshots.
+Keep in mind you need to make your PRs to the dev branch.
```
Comment (AbhijithGanesh) [inline Contributing.md:45]: Can you add it in bold ?

Comment (AbhijithGanesh) [inline Contributing.md:45]:
```suggestion
*Keep in mind you need to make your PRs to the dev branch.*
```

**Comment by AbhijithGanesh** [comment]
good work @ari0211 ! This looks good to me!

**Review by FidalMathew — COMMENTED** [review]
Instead of saying "Star the repo" here. Present this in Readme.md. There are some grammatical mistakes, take note of it.

**Comment by Arjun31415** [comment]
Guys clearly this project uses npm 
why does it say yarn

**Pull request merged**